### PR TITLE
Create Controllers

### PR DESCRIPTION
**Pull request by AzamatG17**

### CODE REVIEW

File: BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            return CreatedAtAction(nameof(Get), new { id = analysisType.Id }, analysisType);
+        }
+
+        [HttpPut]
+        public ActionResult Put(int id,
+            [FromBody] AnalysisTypeForUpdateDTO analysisTypeForUpdateDTO)
+        {
+            _analysisTypeService.UpdateAnalysisType(analysisTypeForUpdateDTO);
+
+            return NoContent();
```
Comment (khojianvar) [inline BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs:55]: in the PUT method it is worth showing the result, as in the POST method

File: BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            };
+        }
+
+        class PagenationMetaData
+        {
+            public int Totalcount { get; set; }
+            public int PageSize { get; set; }
+            public int CurrentPage { get; set; }
+            public int TotalPages { get; set; }
+        }
```
[the remark this answers is not in the document]
Comment (AzamatG17) [reply BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs:85]: Yes, I fixed it, we can look it from the beginning

**Review by khojianvar — APPROVED** [review]

File: BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (khojianvar) [reply BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs:85]: that`s good

File: BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+namespace BioMed.Api.Controllers
+{
+    [Route("api/analysisType")]
+    [ApiController]
+    public class AnalysisTypeController : Controller
+    {
+        private readonly IAnalysisTypeService _analysisTypeService;
+        public AnalysisTypeController(IAnalysisTypeService analysisTypeService)
+        {
+            _analysisTypeService = analysisTypeService;
```
[the remark this answers is not in the document]
Comment (AzamatG17) [reply BioMed.Api/BioMed.Api/Controllers/AnalysisTypeController.cs:19]: not necessarily

**Pull request merged**